docs/earthly-command/earthly-command.md

```diff
@@ -695,3 +695,91 @@ Prints help information about earthly.
 #### Description
 
 Prints version information about earthly.
+
+## earthly ls
+
+#### Synopsis
+
+* ```
+  earthly ls [<project-ref>]
+  ```
+
+#### Description
+
+Prints all targets in an `Earthfile` in a project.
+
+#### Options
+
+##### `--args`
+
+Show arguments (`ARG` statements) in the targets.
+
+##### `--long`
+
+Show full target-ref, including project-ref.
```

Comment:
Show full, canonical, target references (includes the project part of the reference, if applicable).